tests/boulder-integration.sh

```diff
@@ -84,6 +84,36 @@ if [ "$size1" -lt 3000 ] || [ "$size2" -lt 3000 ] || [ "$size3" -gt 1800 ] ; the
     exit 1
 fi
 
+# ECDSA
+openssl ecparam -genkey -name secp384r1 -out "${root}/privkey-p384.pem"
+SAN="DNS:ecdsa.le.wtf" openssl req -new -sha256 \
+    -config "${OPENSSL_CNF:-openssl.cnf}" \
+    -key "${root}/privkey-p384.pem" \
+    -subj "/" \
+    -reqexts san \
+    -outform der \
+    -out "${root}/csr-p384.der"
+common auth --csr "${root}/csr-p384.der" \
+    --cert-path "${root}/csr/cert-p384.pem" \
+    --chain-path "${root}/csr/chain-p384.pem"
+openssl x509 -in "${root}/csr/cert-p384.pem" -text
+openssl x509 -in "${root}/csr/chain-p384.pem" -text
+
+# OCSP Must Staple
+openssl genrsa -out "${root}/privkey-must-staple.pem" 2048
+SAN="DNS:must-staple.le.wtf" openssl req -new -sha256 \
+    -config examples/openssl-must-staple.cnf \
+    -key "${root}/privkey-must-staple.pem" \
+    -subj "/" \
+    -reqexts san \
+    -out "${root}/csr-must-staple.pem"
+common auth --must-staple \
+    --csr "${root}/csr-must-staple.pem" \
+    --cert-path "${root}/csr/cert-must-staple.pem" \
+    --chain-path "${root}/csr/chain-must-staple.pem"
+openssl x509 -in "${root}/csr/cert-must-staple.pem" -text | grep '1.3.6.1.5.5.7.1.24'
```

Comment:
@jsha I get the same result. 
Upgraded from `OpenSSL 1.0.2g-fips  1 Mar 2016` to `OpenSSL 1.0.2h  3 May 2016`. I'm on ubuntu 16.04.

```
X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication, TLS Web Client Authentication
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Subject Key Identifier: 
                17:6B:EA:84:4D:41:21:61:DD:C1:82:08:66:23:BA:0B:24:74:22:F7
            X509v3 Authority Key Identifier: 
                keyid:FB:78:4F:12:F9:60:15:83:2C:9F:17:7F:34:19:B3:2E:36:EA:41:89

            Authority Information Access: 
                OCSP - URI:http://127.0.0.1:4002/
                CA Issuers - URI:http://127.0.0.1:4000/acme/issuer-cert

            X509v3 Subject Alternative Name: 
                DNS:must-staple.le.wtf
            X509v3 CRL Distribution Points: 

                Full Name:
                  URI:http://example.com/crl

            1.3.6.1.5.5.7.1.24: 
                0....
            X509v3 Certificate Policies: 
                Policy: 2.23.140.1.2.1
                Policy: 1.2.3.4
                  CPS: http://example.com/cps
                  User Notice:
                    Explicit Text: Do What Thou Wilt
```